src/Microsoft.AspNet.OData.Shared/Formatter/EdmLibHelpers.cs

```diff
@@ -161,6 +164,18 @@ private static IEdmType GetEdmType(IEdmModel edmModel, Type clrType, bool testCo
                     clrType = underlyingType;
                 }
 
+                if (_cache.TryGetValue(edmModel, out var cachedClrTypes))
```

Comment:
Please use ConcurrentDictionary<Type, IEdmType> instead of var

Reply:
ack!

Reply:
Done.